.coveralls.yml

```diff
@@ -1,3 +1,3 @@
-service_name: travis-ci
+service_name: github-actions
```

Comment:
I'm not 100% about this change. Obviously we won't be using Travis anymore. Coveralls mentions this can be "any string" but only documents travis-ci or travis-pro to be proper values. So unsure what to do here.

https://github.com/php-coveralls/php-coveralls#coverallsyml

Reply:
Can you please remove coveralls (dependency and config file)? Taking the opportunity for some clean-up :)